backend/infrahub/core/account.py

```diff
@@ -145,24 +145,93 @@ async def query_init(self, db: InfrahubDatabase, **kwargs: Any) -> None:
         WITH account, r1 as r
         WHERE r.status = "active"
         WITH account
-        MATCH group_path = (account)-[]->(:Relationship {name: "group_member"})
-            <-[]-(:%(account_group_node)s)
-            -[]->(:Relationship {name: "role__accountgroups"})
-            <-[]-(:%(account_role_node)s)
-            -[]->(:Relationship {name: "role__permissions"})
-            <-[]-(object_permission:%(object_permission_node)s)
-            -[:HAS_ATTRIBUTE]->(:Attribute {name: "branch"})
-            -[:HAS_VALUE]->(object_permission_branch:AttributeValue)
+        CALL {
+            WITH account
+            MATCH (account)-[r1:IS_RELATED]->(:Relationship {name: "group_member"})<-[r2:IS_RELATED]-(account_group:%(account_group_node)s)
```

Comment:
you might want to remove the directionality in all of these queries unless you are absolutely certain that they will always be correct (`()-[]-()` instead of `()-[]->()`)